### ISSUE TITLE
 cannot unmarshal string into Go struct field Edged.volumeStatsAggPeriod of type time.Duration

### ISSUE THREAD

**Issue by darunshen** [kubeedge/kubeedge#1694]
**What happened**:
/var/log/kubeedge/edgecore.log have error: 
```
E0518 20:51:28.697282    1960 helper.go:34] Failed to unmarshal configfile /etc/kubeedge/config/edgecore.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go struct field Edged.volumeStatsAggPeriod of type time.Duration
F0518 20:51:28.697383    1960 server.go:64] error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go struct field Edged.volumeStatsAggPeriod of type time.Duration

```
**What you expected to happen**:
should not print this error
**How to reproduce it (as minimally and precisely as possible)**:
execute 'keadm join ...',then look at the log file
**Anything else we need to know?**:
kubeedge/keadm/cmd/keadm/app/cmd/util/common.go:

**EdgeSide Environment**:
- edgecore version (e.g. `edgecore --version`): v1.3.0-dirty
- Hardware configuration (e.g. `lscpu`):aarch64
- OS (e.g. `cat /etc/os-release`):Ubuntu 18.04.3 LTS
- Kernel (e.g. `uname -a`):4.4.194
- Go version (e.g. `go version`): 1.14.2 linux/arm64

**Comment by zzxgzgz** [comment]
Same problem here. 

I looked into the code, it is trying to parse this field in the config file("1m0s") into a time.Minute(60000000000), I think it is an error caused by type mismatch.

**Comment by darunshen** [comment]
> Same problem here.
> 
> I looked into the code, it is trying to parse this field in the config file("1m0s") into a time.Minute(60000000000), I think it is an error caused by type mismatch.

Maybe .
Everytime running `'keadm join ...`, the `/etc/kubeedge/config/edgecore.yaml` will be recreated, and `volumestatsaggperiod`'s default value `1m0s` is written , then cause the error above by edgecore.

**Comment by sanbaideng** [comment]
same problem in kebeedge v1.3.0

**Comment by zzxgzgz** [comment]
> > Same problem here.
> > I looked into the code, it is trying to parse this field in the config file("1m0s") into a time.Minute(60000000000), I think it is an error caused by type mismatch.
> 
> Maybe .
> Everytime running `'keadm join ...`, the `/etc/kubeedge/config/edgecore.yaml` will be recreated, and `volumestatsaggperiod`'s default value `1m0s` is written , then cause the error above by edgecore.

A work around for that is to run edgecore following the [Development Guild](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/develop_kubeedge.md#run-kubeedge), and run it after manually changing this field in the  /etc/kubeedge/config/edgecore.yaml from "1m0s" to 60000000000.

**Comment by yzhao66** [comment]
> > > Same problem here.
> > > I looked into the code, it is trying to parse this field in the config file("1m0s") into a time.Minute(60000000000), I think it is an error caused by type mismatch.
> > 
> > 
> > Maybe .
> > Everytime running `'keadm join ...`, the `/etc/kubeedge/config/edgecore.yaml` will be recreated, and `volumestatsaggperiod`'s default value `1m0s` is written , then cause the error above by edgecore.
> 
> A work around for that is to run edgecore following the [Development Guild](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/develop_kubeedge.md#run-kubeedge), and run it after manually changing this field in the /etc/kubeedge/config/edgecore.yaml from "1m0s" to 60000000000.
can you write the process？

> > > Same problem here.
> > > I looked into the code, it is trying to parse this field in the config file("1m0s") into a time.Minute(60000000000), I think it is an error caused by type mismatch.
> > 
> > 
> > Maybe .
> > Everytime running `'keadm join ...`, the `/etc/kubeedge/config/edgecore.yaml` will be recreated, and `volumestatsaggperiod`'s default value `1m0s` is written , then cause the error above by edgecore.
> 
> A work around for that is to run edgecore following the [Development Guild](https://github.com/kubeedge/kubeedge/blob/master/docs/setup/develop_kubeedge.md#run-kubeedge), and run it after manually changing this field in the /etc/kubeedge/config/edgecore.yaml from "1m0s" to 60000000000.

Can you write the process?

**Comment by fisherxu** [comment]
The issue has been fixed in #1701, for workaround you can compile a keadm from master branch, and we will release the fix in 1.3.1
